src/sentry/sentry_metrics/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 Module for the Metrics product built by sentry.
-Not to be confused with sentry.metrics, which collects statsd / datadog metrics
+Not to be confused with sentry.metrics, which collects stats / datadog metrics
```

Comment:
Was this change intentional? `statsd` isn't a typo as far as I know.

Reply:
oh, I thought it was a typo....will revert it...thanks!